test_runner/regress/test_lsn_mapping.py

```diff
@@ -45,23 +45,89 @@ def test_lsn_mapping(neon_env_builder: NeonEnvBuilder):
         # Check edge cases: timestamp in the future
         probe_timestamp = tbl[-1][1] + timedelta(hours=1)
         result = client.timeline_get_lsn_by_timestamp(
-            env.initial_tenant, new_timeline_id, f"{probe_timestamp.isoformat()}Z"
+            env.initial_tenant, new_timeline_id, f"{probe_timestamp.isoformat()}Z", 1
         )
         assert result == "future"
 
         # timestamp too the far history
         probe_timestamp = tbl[0][1] - timedelta(hours=10)
         result = client.timeline_get_lsn_by_timestamp(
-            env.initial_tenant, new_timeline_id, f"{probe_timestamp.isoformat()}Z"
+            env.initial_tenant, new_timeline_id, f"{probe_timestamp.isoformat()}Z", 1
         )
         assert result == "past"
 
         # Probe a bunch of timestamps in the valid range
         for i in range(1, len(tbl), 100):
             probe_timestamp = tbl[i][1]
             lsn = client.timeline_get_lsn_by_timestamp(
-                env.initial_tenant, new_timeline_id, f"{probe_timestamp.isoformat()}Z"
+                env.initial_tenant, new_timeline_id, f"{probe_timestamp.isoformat()}Z", 1
```

Comment:
These make it seem that there are three versions:
- default (0 used with unwrap_or)
- 1 (used here)
- 2 (used in the other test case)

But.. Maybe this does not matter for the one release cycle we need to keep this.

Reply:
Please confirm that I understood correctly that there's a hidden "versions 0 and 1 are the same"?

Reply:
yeah the two versions 0 and 1 are the same. Had I chosen 0 as the start I'd probably have gotten feedback how version 1 is actually version 2 in accordance to how you usually use version numbers...

This is an integer only for the case where we decide in the future to *not* remove the param, which I doubt we will, but in that instance it's better than a bool.